posts/2023-08-03-23.0.0.8-beta.adoc

```diff
@@ -0,0 +1,140 @@
+blog_description: Open Liberty 23.0.0.8-beta introduces support for Spring Boot 3, enabling developers to take advantage of utilizing Spring Boot 3.x function within their Liberty applications.  This beta release also includes the very early preview of Jakarta Data.  
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+open-graph-image-alt: Open Liberty Logo
+---
+= Spring Boot 3 support in Open Liberty 23.0.0.8-beta
+Michal Broz <https://github.com/mbroz2>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+Open Liberty 23.0.0.8-beta introduces support for Spring Boot 3, so you can use Spring Boot 3.x function in your Liberty applications.  This beta release also includes the very early preview of Jakarta Data.  
+
+The link:{url-about}[Open Liberty] 23.0.0.8-beta includes the following beta features (along with link:{url-prefix}/docs/latest/reference/feature/feature-overview.html[all GA features]):
+
+* <<sp3, Spring Boot 3.0 support>>
+* <<data, Early preview of Jakarta Data>>
+
+See also link:{url-prefix}/blog/?search=beta&key=tag[previous Open Liberty beta blog posts].
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // // 
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/25679
+// Contact/Reviewer: ReeceNana,hlhoots
+// // // // // // // // 
+[#sp3]
+== Spring Boot 3.0 support
+
+The Liberty Spring Boot Support 3.0 feature provides more complete support for running a Spring Boot 3.0 application on Liberty as well as having the capability to "thin" the application when creating applications in containers. 
+
+Prior releases of Liberty provided support for Spring Boot 1.5 and Spring Boot 2.0 applications.  In the 23.0.0.3 release, support was extended to include link:/blog/2023/06/15/running-spring-boot-3.html[Running a Spring Boot 3.x application WAR file on Liberty].  With this new Spring Boot 3.0 feature, users now have support for both `.wars` and `.jars` by enabling the `springboot-3.0` feature.
+
+To run a Spring Boot 3.x application on Liberty, you must be running with Java 17 or higher.  Additionally, if your application uses the link:/docs/latest/reference/feature/servlet.html[Jakarta Servlet] feature, it must be Jakarta Servet 6.0.  The features can be configured via the `server.xml` file as follows:
+
+[source,xml]
+----
+<features>
+   <feature>springboot-3.0</feature>
+   <feature>servlet-6.0</feature>
+</features>
+----
+
+As with previous versions of the Spring Boot feature, specify link:/docs/latest/reference/config/springBootApplication.html[Spring Boot configuration elements] in the `server.xml` file. For example:
+
+[source,xml]
+----
+ <springBootApplication id="spring-boot-app" location="spring-boot-app-0.1.0.jar" name="spring-boot-app" />
+----
+
```

Comment:
Similar to previous versions, SB application jar's can use the /dropins/spring folder, and skip server.xml entries.